react/components/ReceiverInfoForm.tsx

```diff
@@ -31,7 +32,10 @@ const ReceiverInfoForm: React.FC<Props> = ({
 
   return (
     <form onSubmit={handleSubmit}>
-      <span className="fw6 dib mb6">
+      <div className="c-muted-1 lh-copy">
+        <PlaceDetails display="extended" />
+      </div>
```

Comment:
WDYT of having a label like "Complete your address" here just like the "previous" step? might be worth asking the designers

Reply:
@augustocb What do you think? In the previous step there's this "Complete your address" label.
<p align=center><img src="https://user-images.githubusercontent.com/8902498/86160733-91d12700-bae2-11ea-9fdc-28670c4219c0.png" width=400 /></p>

Do we keep this label on the receiver name screen or keep it as it is shown in the PR description?